uint32 auction_round
Task[] tasks
